app/assets/javascripts/application.js.coffee

```diff
@@ -197,3 +197,9 @@ $ ->
     new ConfirmDangerModal(form, text)
 
   new Aside()
+
+  $('.hide-project-limit-message').on 'click', (e) ->
+    path = '/'
+    $.cookie('hide_project_limit_message', 'false', { path: path })
+    $(@).parents('.project-limit-message').remove()
+    e.preventDefault()
```

Comment:
This code obviously doesn't belong in `application.js.coffee` but I wasn't sure where it should go that it would actually be executed.